src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderMovementApplication.kt

```diff
@@ -118,12 +118,13 @@ class OffenderMovementApplication(
 
   @Column(name = "COMMENT_TEXT")
   val comment: String? = null,
-//
-//  @Column(name = "TO_ADDRESS_OWNER_CLASS")
-//  val toAddressOwnerClass: String? = null,
-//
-//  @Column(name = "TO_ADDRESS_ID")
-//  val toAddressId: Long? = null,
+
+  @Column(name = "TO_ADDRESS_OWNER_CLASS")
+  val toAddressOwnerClass: String? = null,
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "TO_ADDRESS_ID")
+  val toAddress: Address? = null,
```

Comment:
The main change is to support addresses. We can't map the address owner class from the Address because we expect owner class of AGY, CORP or OFF but the OFFENDER_MOVEMENT_APPS table has loads where the address owner class is actually the AGY_LOC_ID. I'll deal with them next.